dom.bs

```diff
@@ -1488,7 +1488,7 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}},
+<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}}, {{CDATASection}},
```

Comment:
Hmm, so I suppose that this is then enumerating the least-derived interfaces that can be instantiated? Otherwise it'd collapse into just saying "Node" :)